src/patternfly/components/Wizard/wizard.scss

```diff
@@ -270,7 +271,6 @@
   flex-wrap: wrap;
   align-items: baseline;
   margin-right: var(--pf-c-wizard__toggle-list--MarginRight);
-  margin-bottom: var(--pf-c-wizard__toggle-list--MarginBottom);
```

Comment:
Then add this back if you add `--pf-c-wizard__toggle-list--MarginBottom` back.

Reply:
Done.